### PR TITLE
Server shinan

### PR DESCRIPTION
**Pull request by ShinanSun**
create carousels for Home page, I have changed quite a bit in the home page. please review.

### CODE REVIEW

File: client/components/homePage/bookModal.jsx [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { Modal, Image, Button, Row, Col } from 'react-bootstrap';
+
+//Props: id from Google books (number), show & owned & inLibrary (boolean), handleClose & handleAddToLibrary & handlePurchase functions
+export default function BookModal({ book, show, setShow }) {
+  console.log('waht is clicked book', book)
```
Comment (jfleming9357) [inline client/components/homePage/bookModal.jsx:6]: Lets keep console logs off of the main branch

**Review by jfleming9357 — APPROVED** [review]
The homepage looks fantastic!

File: client/context/context.jsx [hunk 2]
```diff
@@ -132,6 +146,10 @@ export const AppProvider = ({ children }) => {
   return (
     <AppContext.Provider
       value={{
+<<<<<<< HEAD
```
Comment (jfleming9357) [inline client/context/context.jsx:149]: Oops thought I saw a check running and thought this got resolved. Fix this any you can merge

**Review by jfleming9357 — CHANGES_REQUESTED** [review]

**Review by daniel-he-dev — CHANGES_REQUESTED** [review]

**Comment by ShinanSun** [comment]
Yes Sir!

File: client/components/homePage/books.jsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import BookCarosel from './booksCarosel.jsx'
```
[the remark this answers is not in the document]
Comment (ShinanSun) [reply client/components/homePage/books.jsx]: sorry, that's so embarrassing

File: client/components/homePage/bookModal.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (ShinanSun) [reply client/components/homePage/bookModal.jsx]: removed

File: client/context/context.jsx (hunk 2 above)
Comment (ShinanSun) [reply client/context/context.jsx]: thank u for checking carefully. Had a rush morning when sending PR.

**Pull request merged**